.github/CONTRIBUTING.md

```diff
@@ -31,70 +31,93 @@ If you at any point have any questions please read: [I have a question related t
-  - **Request that the pull-request is [UX reviewed](#ux-review)**
-  - **Get your pull-request merged** - when the pull-request has been approved it will automatically be merged to master via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).
-5. **Celebrate! 🎉** - your changes have been added to Kirby!
-
-### I have an enhancement 
-- Submit [new issue](https://github.com/kirbydesign/designsystem/issues/new/choose) using the enhancement request template
-  - please read "[the good: issue](#issue)"
-- Contact kirby-guild to get it tech refined 
-- If you would like to implement issue: 
-  -please read: 
-    - "[Setup environment and tooling for contribution](#setup-environment-and-tooling-for-contribution)" to get properly setup 
-    - "[Creating pull-requests](#creating-pull-requests)" to get informed on how to most easily get your changes merged
-  - assign yourself to the issue to signal to others you are working on it
-  - if you at any point have questions while solving the issue please see: [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
-
-### I would like to report a bug 
-- Submit [new issue](https://github.com/kirbydesign/designsystem/issues/new/choose) using the bug template  
-  - please read "[the good: issue](#issue)"
-- Contact kirby-guild to get it verified 
-  - we verify because it might be a feature not a bug 🐛
-- If you would like to implement issue: 
-  -please read: 
-    - "[Setup environment and tooling for contribution](#setup-environment-and-tooling-for-contribution)" to get properly setup 
-    - "[Creating pull-requests](#creating-pull-requests)" to get informed on how to most easily get your changes merged
-  - assign yourself to the issue to signal to others you are working on it
-  - if you at any point have questions while solving the issue please see: [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
+1. **Open a [new issue](https://github.com/kirbydesign/designsystem/issues/new/choose) using the "component request"-template** (See guidelines on how to write [the good issue](#issue)).
+2. **Refinement:**
+    - Request that the issue is [UX refined](#ux-refinement) in [#kirby-guild](#getting-access-to-the-kirby-guild-slack-channel); do not proceed until this is done.
+    - Request that the issue is [tech refined](#tech-refinement) in [#kirby-guild](#getting-access-to-the-kirby-guild-slack-channel); do not proceed until this is done.
+    - Move the issue to the "Ready to do"-column in the "Kirby kan-ban board"-project.
+3. **Implementation:**
+    - Make sure you have read: [before you get coding](#before-you-get-coding).
+    - Signal to others you are working on the issue by assigning yourself.
+    - Create a "feature"-branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master). 
```

Comment:
Should we add a section about branch naming conventions that we can link to here?

Reply:
A "Branch" heading could be added to "The good" section :D

Reply:
I have added a "the good: branch" section to the outline: #1404

Reply:
Great. Thanks. I won't add it then 😁